diagnostic.md

```diff
@@ -5,20 +5,24 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    nested routes
```

Comment:
This is unclear, can you expand on what you mean?  Are you referring to a navbar w/ buttons?  Other examples would be; a list w/ list-items, a sidebar for a blog w/ comments... etc.